models.py

```diff
@@ -56,8 +55,40 @@ class User (db.Model):
         db.Text,
         nullable=True
     )
+    
     review = db.relationship('Review', backref='users')
 
+    @classmethod
+    def signup(cls, firstname, lastname, username, email, password, image, state, vax_date, covid_status):
+        """Sign up user. Hashes password and adds user to system."""
+        hashed_pwd = bcrypt.generate_password_hash(password).decode('UTF-8')
+
+        user = User(
+            firstname=firstname,
+            lastname=lastname,
+            username=username,
+            email=email,
+            password=hashed_pwd,
+            image=image,
+            state=state,
+            vax_date=vax_date,
+            covid_status=covid_status)
+
+        db.session.add(user)
+        return user
```

Comment:
@azaria-dedmon how is this related to login/logout ?

Reply:
I have to include the signup functionality in order to properly test for the login/logout feature. To test if a user is able to login/logout, I have to first register an account.